src/plugins/intel_gpu/src/kernel_selector/kernels/mvn/mvn_kernel_bfyx_opt.cpp

```diff
@@ -85,30 +85,29 @@ JitConstants MVNKernelBfyxOpt::GetJitConstants(const mvn_params& params, MVNKern
         auto w = toCodeString(input.W(), 2);
         auto f = toCodeString(input.Feature(), 1);
         auto b = toCodeString(input.Batch(), 0);
-        auto multiply = [](std::vector<std::string> dims) -> std::string {
-            std::string res = "(";
-            for (size_t i = 0; i < dims.size(); i++) {
-                auto& d = dims[i];
-                res += d;
-                if (i != dims.size() - 1)
-                    res += "*";
-            }
-            res += ")";
-            return res;
-        };
         std::string data_set_size;
         std::string data_set_count;
         if (params.mvnMode == MVNMode::WITHIN_CHANNELS) {
-            data_set_size = multiply({x, y, z});
-            data_set_count = multiply({f, b});
+            data_set_size = toVectorMulString({x, y, z});
+            data_set_count = toVectorMulString({f, b});
         } else {
-            data_set_size = multiply({x, y, z, f});
+            data_set_size = toVectorMulString({x, y, z, f});
             data_set_count = b;
         }
+        // since lws[0] is calculated by power of 2
+        // items_num can be calculated by dividing data_set_size by power of 2
+        const std::string lws_0 = "get_local_size(0)";
+        const std::string power = "(uint)(log2((float)" + lws_0 + "))";
```

Comment:
Please do not use log function in kernel. It  is expensive..

Reply:
replaced built-in log2 function with function macro